### PR TITLE
LPS-196565 Backend tests

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPS-196565

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-test/src/com/liferay/portal/kernel/test/portlet/MockLiferayPortletURL.java [hunk 1]
```diff
@@ -338,4 +341,85 @@ public void write(Writer writer, boolean escapeXML) throws IOException {
...
+			return false;
+		}
+
+		@Override
+		public boolean isPublic(String s) {
+			return false;
+		}
+
+		@Override
+		public boolean removeParameter(String s) {
```
Comment (ealonso) [inline portal-test/src/com/liferay/portal/kernel/test/portlet/MockLiferayPortletURL.java:397]: @lfbesada I think we should replace **s** with **name**

File: portal-test/src/com/liferay/portal/kernel/test/portlet/MockLiferayPortletURL.java [hunk 2]
```diff
@@ -338,4 +341,85 @@ public void write(Writer writer, boolean escapeXML) throws IOException {
...
+			return this;
+		}
+
+		@Override
+		public String setValue(String s, String s1) {
+			return null;
+		}
+
+		@Override
+		public String[] setValues(String s, String... strings) {
```
Comment (ealonso) [inline portal-test/src/com/liferay/portal/kernel/test/portlet/MockLiferayPortletURL.java:414]: **strings** should be **names**

**Comment by lfbesada** [comment]
@ealonso Thanks for catching them :heart: I have fixed them. Regards

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/layout/layout-page-template-admin-web-test/src/testIntegration/java/com/liferay/layout/page/template/admin/web/internal/portlet/action/test/UpdateDisplayPageEntryContentTypeMVCActionCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,463 @@
...
+		return _ddmStructureLocalService.addStructure(
+			TestPropsValues.getUserId(), _group.getGroupId(), 0,
+			_portal.getClassNameId(JournalArticle.class.getName()), null,
+			RandomTestUtil.randomLocaleStringMap(), null, ddmForm,
+			ddmFormLayout, StorageType.DEFAULT.toString(),
+			DDMStructureConstants.TYPE_DEFAULT, serviceContext);
+	}
+
+	private void _assertLayoutPageTemplateEntry(
+			long classNameId, long classTypeId,
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web-test/src/testIntegration/java/com/liferay/layout/page/template/admin/web/internal/portlet/action/test/UpdateDisplayPageEntryContentTypeMVCActionCommandTest.java:305]: classTypeId is always 0, I think you can inline it or just remove it

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
@ealonso Feedback added! Thanks a lot!

**Pull request closed without merging**